File: public/example/example_1.cs
using System;

class SimpleCSharpExample
{
    static int Multiply(int x, int y)
    {
        return x * y;
    }

    static void Main()
    {
        int num1 = 5;
        int num2 = 10;
        int product = Multiply(num1, num2);
        Console.WriteLine($"The product is: {product}");
    }
}

File: public/example/example_1.c
#include <stdio.h>

void swap(int *x, int *y) {
    int temp = *x;
    *x = *y;
    *y = temp;
}

int main() {
    int num1 = 5, num2 = 10;
    
    printf("Before swap: num1 = %d, num2 = %d\n", num1, num2);
    
    swap(&num1, &num2);
    
    printf("After swap: num1 = %d, num2 = %d\n", num1, num2);

    return 0;
}
